msg/vehicle_attitude.msg

```diff
@@ -4,4 +4,4 @@ float32 pitchspeed	# Angular velocity about body east axis (y) in rad/s
 float32 yawspeed	# Angular velocity about body down axis (z) in rad/s
 float32[4] q		# Quaternion (NED)
 
-# TOPICS vehicle_attitude vehicle_attitude_groundtruth
+# TOPICS vehicle_attitude vehicle_attitude_groundtruth vehicle_vision_attitude
```

Comment:
Mocap will be going into `vehicle_attitude(/position)_groundtruth`, so its not an issue.